config/default/core.entity_form_display.node.judgement.default.yml

```diff
@@ -0,0 +1,357 @@
+uuid: d6308cb8-4066-40f5-934b-43d0c2622688
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.judgement.body
+    - field.field.node.judgement.field_case_history
+    - field.field.node.judgement.field_casename
+    - field.field.node.judgement.field_casenumber
+    - field.field.node.judgement.field_cases_considered
+    - field.field.node.judgement.field_court_name
+    - field.field.node.judgement.field_external_link
+    - field.field.node.judgement.field_files
+    - field.field.node.judgement.field_flynote
+    - field.field.node.judgement.field_flynote_sync_local
+    - field.field.node.judgement.field_headnote_and_holding
+    - field.field.node.judgement.field_jno
+    - field.field.node.judgement.field_judge
+    - field.field.node.judgement.field_legislation_considered
```

Comment:
This isn't in the spreadsheet?

Reply:
They are are located under metadata?

Reply:
Oh, I see `field_legislation` in the spreadsheet now. I'm not sure how Neil wants to use this.